### PR TITLE
Add Helm Chart

### PR DESCRIPTION
**Pull request by 1995parham**
I have added a simple helm chart for having the surveyor on Kubernetes.

### CODE REVIEW

File: chart/surveyor/values.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+tolerations: []
+
+affinity: {}
+
+config:
+  targets: "nats://127.0.0.1:4222"
+  creds: |
+    -----BEGIN NATS USER JWT-----
+    eyJ0eXAiOiJqd3QiLCJhbGciOiJlZDI1NTE5In0.eyJqdGkiOiJUVlNNTEtTWkJBN01VWDNYQUxNUVQzTjRISUw1UkZGQU9YNUtaUFhEU0oyWlAzNkVMNVJBIiwiaWF0IjoxNTU4MDQ1NTYyLCJpc3MiOiJBQlZTQk0zVTQ1REdZRVVFQ0tYUVM3QkVOSFdHN0tGUVVEUlRFSEFKQVNPUlBWV0JaNEhPSUtDSCIsIm5hbWUiOiJvbWVnYSIsInN1YiI6IlVEWEIyVk1MWFBBU0FKN1pEVEtZTlE3UU9DRldTR0I0Rk9NWVFRMjVIUVdTQUY3WlFKRUJTUVNXIiwidHlwZSI6InVzZXIiLCJuYXRzIjp7InB1YiI6e30sInN1YiI6e319fQ.6TQ2ilCDb6m2ZDiJuj_D_OePGXFyN3Ap2DEm3ipcU5AhrWrNvneJryWrpgi_yuVWKo1UoD5s8bxlmwypWVGFAA
+    ------END NATS USER JWT------
```
Comment (ripienaar) [inline chart/surveyor/values.yaml:76]: just checking this isnt you'd care to leak out or anything, I'd probably default to a empty one

Comment (1995parham) [reply chart/surveyor/values.yaml:76]: Thanks for mentioning that I have grabbed this example from [here](https://docs.nats.io/developing-with-nats/security/creds).

Comment (ripienaar) [reply chart/surveyor/values.yaml:76]: ok :)

**Comment by ColinSullivan1** [comment]
@wallyqs , should we recommend the author move this work to nats-io/k8s repo?  ...best not to have in two places imo.

**Comment by wallyqs** [comment]
@1995parham would you mind opening up the PR agains the `nats-io/k8s` repo at https://github.com/nats-io/k8s/tree/master/helm/charts/nats ? This way it could be picked up by `helm repo update` and installed like the other ones. Thanks!

**Comment by 1995parham** [comment]
Yes, I am going to change the pull request so, I will close this.

**Pull request closed without merging**